hw/top_earlgrey/dv/top_earlgrey_sim_cfgs.hjson

```diff
@@ -2,7 +2,7 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 {
-  // This is the master cfg hjson for DV simulations. It imports ALL individual DV sim
+  // This is the primary cfg hjson for DV simulations. It imports ALL individual DV sim
```

Comment:
I was going for a terminology for indicating a "group" of individual simulation configurations. I think "group" or "set" ("groupset"?)  or "collection" is perhaps a better term.

Reply:
Turns out "groupset" is a almost exclusively a biking terminology. I'd settle for "group". 
```suggestion
  // This is a cfg hjson group for DV simulations. It includes ALL individual DV simulation
```

Reply:
> I was going for a terminology for indicating a "group" of individual simulation configurations. I think "group" or "set" ("groupset"?) or "collection" is perhaps a better term.

I'll update with this terminology, lmk if I caught them all after the next push.